torch/nn/modules/module.py

```diff
@@ -357,6 +359,7 @@ def _slow_forward(self, *input, **kwargs):
     def __call__(self, *input, **kwargs):
         for hook in self._forward_pre_hooks.values():
             hook(self, input)
+        import torch.jit
```

Comment:
Previously it was ok without those imports. Why are they necessary now?

Reply:
I'm not entirely sure. I think it was the case that by the time this runs, something else had caused torch.jit to be imported, and it's no longer the case. I'll dig into it a bit more